pkg/storage/unified/search/bleve.go

```diff
@@ -314,7 +314,7 @@ func (b *bleveBackend) BuildIndex(
 			defer closeIndexOnExit(index, indexDir) // Close index, and delete new index directory.
 		}
 	} else {
-		index, err = bleve.NewMemOnly(mapper)
+		index, err = bleve.NewUsing("", mapper, bleve.Config.DefaultIndexType, bleve.Config.DefaultMemKVStore, nil)
```

Comment:
See https://github.com/blevesearch/bleve/blame/master/index.go#L301 for comparison

Reply:
I would suggest to change `bleve.New` call on line 302 as well, so that it's clear that they both use the same setting. Perhaps we could have `newBleveIndex(path)` function to hide this.